indico/modules/events/abstracts/blueprint.py

```diff
@@ -16,11 +16,12 @@
 
 from __future__ import unicode_literals
 
-from indico.modules.events.abstracts.controllers.management import RHAbstracts
+from indico.modules.events.abstracts.controllers.management import RHAbstracts, RHAbstractList
 from indico.web.flask.wrappers import IndicoBlueprint
 
 _bp = IndicoBlueprint('abstracts', __name__, url_prefix='/event/<confId>', template_folder='templates',
                       virtual_template_folder='events/abstracts')
 
 # Management
-_bp.add_url_rule('/manage/abstracts/', 'manage_abstracts', RHAbstracts)
+_bp.add_url_rule('/manage/abstracts/setup', 'manage_abstracts', RHAbstracts)
```

Comment:
But for registrations it's not the top-level thing:

`/manage/registration/<int:reg_form_id>/registrations/`


If you want to go the same way I'd rather use `/manage/abstracts/abstracts/` - otherwise it'll be a mess, especially with the endpoints for the various settings dialogs